uint32 SRR_SENSOR_ID=0
uint32 LRR_SENSOR_ID=1
uint32 CAM_SENSOR_ID=2

uint32 obj_id
float64 x
float64 y
float64 timestamp
